coast/gridded.py

```diff
@@ -0,0 +1,865 @@
+import os.path as path_lib
+import warnings
+
+# from dask import delayed, compute, visualize
+# import graphviz
+import gsw
+import numpy as np
+import xarray as xr
+
+from . import general_utils, stats_util
+from .COAsT import COAsT
+from .config import ConfigParser
+from .logging_util import get_slug, debug, info, warn, error
+
+
+class Gridded(COAsT):  # TODO Complete this docstring
+    """
+    Words to describe the NEMO class
+
+    kwargs -- define addition keyworded arguemts for domain file. E.g. ln_sco=1
+    if using s-scoord in an old domain file that does not carry this flag.
+    """
+
+    def __init__(self, fn_data=None, fn_domain=None,  # TODO Super init not called + add a docstring
```

Comment:
Is there a reason we don't get around to calling the init in this version? Breaking inheritance like this is a bit misleading for users.

Reply:
Maybe @jpolton or @davbyr can help here, if we call an empty super __init__ it will output a message saying we are not loading a file (which could be miss leading) if we pass a file the super method would go off and load the file which we dont want it to do... might be this need a look at in its own ticket